### PR TITLE
Create Database and table

### PR DESCRIPTION
**Pull request by ha-manel**
## In this branch I:
- [x] Created the `vet_clinic` database.
- [x] Created the `animals` table.
- [x] Added queries.

Queries results:
Screenshot 2022-07-06 143959

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '01-01-2016' AND '01-12-2019';
```
Comment (Mihndim2020) [inline queries.sql]: - [ ] I think to include all of 2019, the date_of_birth should be between `01-01-2016 AND 31-12-2019`, please fix the range to match with the requirements(`List the name of all animals born between 2016 and 2019.`).

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @ha-manel, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Clean and organized code
:heavy_check_mark: Good job with Gitflow

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments below  and  under the review._

 #### I will leave the comments here, since your `Readme file` is not accessible for this `PR` as you did not make any change on it ####

- [x] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. It's a good thing you are using the template from Microverse, but please kindly adapt it to your project. Replace all the information in the various sections of the template with information pertaining to you and to your project.  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (ha-manel) [reply queries.sql]: I didn't notice, thanks!

**Review by elmar8287 — APPROVED** [review]
Hi, 

Good job with implementing the suggestions from the previous review!

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**